pages/visualizations/overview/issue_staleness.py

```diff
@@ -190,9 +185,13 @@ def new_staling_issues(repolist, timer_pings, interval, staling_interval, stale_
     # create dataframe from record data
     df = pd.DataFrame(results)
 
-    # change all to datetime
-    df["created"] = pd.to_datetime(df["created"], utc=True)
-    df["closed"] = pd.to_datetime(df["closed"], utc=True)
+    try:
```

Comment:
why these instead of testing above if the results/ df is empty

Reply:
dataframe 'emptiness' is tricky, I found. A dataframe with NaN's will report False when you call 'df.empty', so it's more reliable to just catch the exception returned from trying to cast a non-existent column ('created') to a datetime object than to check for emptiness.

Reply:
Would we be able to check the length/size? Just thinking for simplicity/people creating new visualizations from this structure

Reply:
Yes can experiment with this in the next step, I'll make an issue for it.

Reply:
revolving with issue being opened